plugins/scaffolder-react/src/next/components/TemplateCard/TemplateCard.tsx

```diff
@@ -83,48 +75,32 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
 }));
 
 /**
- * The Props for the Template Card component
+ * The Props for the {@link TemplateCard} component
  * @alpha
  */
 export interface TemplateCardProps {
   template: TemplateEntityV1beta3;
-  deprecated?: boolean;
+  additionalLinks?: {
+    icon: IconComponent;
+    text: string;
+    url: string;
+  }[];
```

Comment:
So these are new features actually. And the removal of `deprecated` is breaking?

Reply:
Nope as they're all under `@alpha` this component is entirely new according to the `-react` package and it wasn't exported from the `@backstage/plugin-scaffolder` beforel.